commonItems/Serialization/PDXSerializer.cs

```diff
@@ -0,0 +1,74 @@
+﻿using ExtensionMethods;
+using System;
+using System.Collections;
+using System.Globalization;
+using System.Reflection;
+using System.Text;
+
+namespace commonItems.Serialization {
+	public static class PDXSerializer {
+		private static readonly CultureInfo cultureInfo = CultureInfo.InvariantCulture;
```

Comment:
This seems not appropriate for a generalized tool.

Reply:
I added this cultureInfo because floating point numbers' string representation is region dependent by default. For me (Polish system version), doubles were being written with `,` instead of `.`, for example `3,14`.
What should I use instead?

Reply:
Ah, I was interpreting that as an in-game culture. Paradox has poisoned my brain.